docs/_docs/focus/validate.md

```diff
@@ -0,0 +1,127 @@
+| `BenefitId`                                                                                                 | `CommitmentDiscountId`                                                          | Avoid using `ReservationId` in Cost Management. Prefer `BenefitId` to include savings plans and map to the fully-qualified ID. Actual cost will be 0 for commitment usage.                                                                                                                                                                 |
+| `BillingPeriodEndDate` + 1 day                                                                              | `BillingPeriodEnd`                                                              | FOCUS uses exclusive end dates (e.g., Feb 1 instead of Jan 31).                                                                                                                                                                                                                                                                            |
+| `BillingPeriodStartDate`                                                                                    | `BillingPeriodStart`                                                            |                                                                                                                                                                                                                                                                                                                                            |
+| `ChargeType` == "Usage"                                                                                     | `ChargeCategory` == "Usage"<br>and `CommitmentDiscountStatus` != "Unused"       | FOCUS includes unused commitments in the usage charge category.                                                                                                                                                                                                                                                                            |
+| `ChargeType` == "Usage"<br>or `ChargeType` == "UnusedReservation"<br>or `ChargeType` == "UnusedSavingsPlan" | `ChargeCategory` == "Usage"                                                     | FOCUS includes unused commitments in the usage charge category.                                                                                                                                                                                                                                                                            |
```

Comment:
@flanakin - In actual cost report, we won't see unused RI or unused ASP. Can we remove them from this table to avoid confusion pls?

Reply:
Good catch!